packages/eventual-send/src/exports.d.ts

```diff
@@ -1,4 +1,4 @@
-import type { HandledPromiseConstructor, EProxy } from './types.d';
+import type { HandledPromiseConstructor } from './types.d';
```

Comment:
Nit: still prefer identifiers with extension
```suggestion
import type { HandledPromiseConstructor } from './types.d.ts';
```